extras/nightly-container/provision.yml

```diff
@@ -0,0 +1,86 @@
+---
+- hosts: builder
+  gather_facts: no
+
```

Comment:
Can we put remote_user or become_method? But default if not run as root yum will
fail.

Or is it that by default it is run as root?

Reply:
It's null by default, so I most likely picks the user the buildah container is being run as. I'll force it to be root.